### ISSUE TITLE
Can you please confirm?

### ISSUE THREAD

**Issue by railsjack** [alexX512/Alexey-Perevyshin-Hash_Map2.0#4]
I am sorry for the abrupt message, but could you please confirm if you ever registered at the codeforce.com ?

I ask this because I chatted with the person in the site, but can't for now.

Thank you.

**Comment by alexX512** [comment]
Yes, I am registered at the Codeforces. If you are Jin, I chatted with you.

**Comment by railsjack** [comment]
Thank you for your reply, my email is jinnahrae@gmail.com and skype is jeannahrae
Can we chat through email or skype?
